src/components/WelcomeScreen.tsx

```diff
@@ -0,0 +1,73 @@
+import { useTexts } from '@lib/TextsContext'
+import Image from 'next/image'
+import { FC } from 'react'
+import introImage from '../images/intro-header.png'
+import stripesPattern from '../images/stripe-pattern.svg'
+import { PrimaryButton } from '@components/PrimaryButton'
+import { SecondaryButton } from '@components/SecondaryButton'
+import { Phone } from '@components/icons/Phone'
+import { useRouter } from 'next/router'
+import classNames from '@lib/classNames'
+import { useIsMobile } from '@lib/hooks/useIsMobile'
+import { LegalFooter } from './LegalFooter'
+
+export const WelcomeScreen: FC<{
+  onShowOffers: () => void
+}> = ({ onShowOffers }) => {
+  const texts = useTexts()
+  const { push } = useRouter()
+  const isMobile = useIsMobile()
+  return (
+    <div>
+      <div
+        className={classNames(
+          `grid grid-cols-1 grid-rows-[auto,auto,1fr,auto,auto,auto]`,
+          isMobile && `min-h-screen`
+        )}
+      >
+        <div>
+          <section className="w-full h-32 md:h-[202px] relative">
+            <Image src={introImage} layout="fill" objectFit="cover" />
+            <span className="absolute right-0 bottom-0 w-20 h-20">
+              <Image
+                {...stripesPattern}
+                alt="decorative pattern"
```

Comment:
Is there a reason for setting the `alt` while it is actually aria-hidden? If not, we could remove it.

Reply:
True, in this case it was eslint complaining about it so I gave it an alt anyway. I can also delete it and add an eslint comment.

Reply:
Yes, in this case I would prefer to ignore eslint, otherwise there is this weird clash between making the pattern more accessible and hiding it from the accessibility tree at the same time.